modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -130,25 +142,20 @@ Only expressions using dots as field separators are allowed. While `\#[payload.p
 | No. +
 Keep in mind that if you don't specify a selector expression, the policy will be applied, but won't tokenize anything from your payload.
 
-| *Format*
+| Format
 | Type of information that will be tokenized. Formats may vary depending on the selected service. Choose one format from the list.
 
-If you do not see any formats in the list, check that the tokenization service selected contains at least one format. +
-See the xref:anypoint-security::tokenization-formats.adoc[Tokenization Formats] article for more information.
+If you do not see any formats in the list, check that the tokenization service that you selected contains at least one
+pppp. 
```

Comment:
typo with 'pppp'?

Reply:
No it was for me as a pointer to come back and check. I was testing a link issue. :-)